addon/components/frost-autocomplete.js

```diff
@@ -315,12 +322,10 @@ export default Component.extend({
         filter: get(selectedItem, 'label'),
         internalSelectedItem: selectedItem
       })
-
-      const onChange = this.get('onChange')
-
+      const {onChange, onChangeSendObject} = this.getProperties(['onChange', 'onChangeSendObject'])
```

Comment:
I believe the array wrapping the properties is unnecessary. Doesn't this work? 
```
const {onChange, onChangeSendObject} = this.getProperties('onChange', 'onChangeSendObject')
```

https://www.emberjs.com/api/ember/release/functions/@ember%2Fobject/getProperties

Reply:
Ya they're equivalent, just following conventions of uses in codebase